RequestService/RequestService.Repo/Helpers/QuestionExtenstions.cs

```diff
@@ -180,6 +180,51 @@ public static void SetQuestionData(this EntityTypeBuilder<Question> entity)
                 AdditionalData = string.Empty,
                 AnswerContainsSensitiveData = true
             });
+
+            entity.HasData(new Question
+            {
+                Id = (int)Questions.GroupSizeAdults,
+                Name = "How many adults need accomodation",
+                QuestionType = (int)QuestionType.Number,
+                AdditionalData = string.Empty,
+                AnswerContainsSensitiveData = false
+            });
+
+            entity.HasData(new Question
+            {
+                Id = (int)Questions.GroupSizeChildren,
+                Name = "How many children need accomodation",
+                QuestionType = (int)QuestionType.Number,
+                AdditionalData = string.Empty,
+                AnswerContainsSensitiveData = false
+            });
+
+            entity.HasData(new Question
+            {
+                Id = (int)Questions.GroupSizePets,
+                Name = "How many pets need accomodation",
+                QuestionType = (int)QuestionType.Text,
+                AdditionalData = string.Empty,
+                AnswerContainsSensitiveData = false
+            });
+
+            entity.HasData(new Question
+            {
+                Id = (int)Questions.PreferredLocation,
+                Name = "Do you have a preferred location within the UK",
+                QuestionType = (int)QuestionType.Radio,
+                AdditionalData = GetAdditionalData(Questions.PreferredLocation),
+                AnswerContainsSensitiveData = false
+            });
+
+            entity.HasData(new Question
+            {
+                Id = (int)Questions.PreferredLanguage,
+                Name = "What languages do you speak?",
```

Comment:
Since we can't do check boxes, I think we go with MVP of a text box currently.

Radio buttons don't allow for e.g. Russian _and_ English.